lib/appsignal/utils/data_encoder.ex

```diff
@@ -31,6 +31,9 @@ defmodule Appsignal.Utils.DataEncoder do
   def encode(resource, {key, value}) when is_binary(value) do
     Nif.data_set_string(resource, key, value)
   end
+  def encode(resource, {key, value}) when is_integer(value) and value >= 9223372036854775808 do
+    Nif.data_set_string(resource, key, "bignum:#{value}")
```

Comment:
Haha, yes, that's what I meant. Fixed in b34b68667444ddacce33074b182397115fffe520. :+1: